### PR TITLE
Update publishing steps

### PR DESCRIPTION
**Pull request by klebba**
Related to #34

### CODE REVIEW

File: deno.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@netflix/eslint-config",
   "version": "3.0.0",
   "imports": {
-    "@eslint/js": "npm:@eslint/js@^8.21.0"
+    "@eslint/js": "npm:@eslint/js@^9.1.1"
```
Comment (klebba) [inline deno.json:5]: Oops, missed this in the ESLint bump

**Comment by klebba** [comment]
@theengineear once we sort this out I'll update the `x-element` and `x-test` docs accordingly

File: PUBLISHING.md [hunk 2]
```diff
@@ -8,21 +8,19 @@ procedure feels more comfortable. By publishing, the “Publish” GitHub Action
...
-   in multiple places to account for the self-package’s version (i.e., `""`).
-   For example, set the `"version"` key to a value of `"1.0.0-rc.57"`.
-2. Add / commit those files. By convention, set the message to the new version
-   (e.g., `git commit --message="1.0.0-rc.57"`).
-3. Tag the commit using an annotated tag. By convention, set the name to the new
-   version _prefixed with a “v”_ and set the message to the new version
-   (e.g., `git tag --annotate "v1.0.0-rc.57" --message="1.0.0-rc.57"`).
-4. Push the new commit / tags by running `git push origin main --follow-tags`.
+1. Edit `package.json` and `deno.json` files to set the new version.
+2. Run `npm install` to derive `package-lock.json` using the new version.
```
Comment (theengineear) [inline PUBLISHING.md:17]: 👌

File: PUBLISHING.md [hunk 3]
```diff
@@ -8,21 +8,19 @@ procedure feels more comfortable. By publishing, the “Publish” GitHub Action
...
+2. Commit the changed files. Follow the convention for the commit message.
```
Comment (theengineear) [inline PUBLISHING.md:18]: Bump numbers here. We have two twos now.

File: package.json [hunk 4]
```diff
@@ -30,16 +30,20 @@
...
   "peerDependencies": {
-    "@eslint/js": "^9.0.0"
+    "@eslint/js": "^9.1.1"
   },
   "devDependencies": {
-    "eslint": "^9.0.0"
+    "eslint": "^9.1.1"
   },
   "scripts": {
-    "test": "ESLINT_USE_FLAT_CONFIG=true eslint --config index.js --debug .",
```
Comment (theengineear) [inline package.json:39]: 🎉

File: package.json [hunk 5]
```diff
@@ -30,16 +30,20 @@
...
+    "test": "eslint --config index.js --debug .",
     "bump": "./bump.sh"
   },
   "contributors": [
+    {
```
Comment (theengineear) [inline package.json:43]: ❤️

**Review by theengineear — DISMISSED** [review]
LGTM! Thanks!

File: PUBLISHING.md (hunk 3 above)
Comment (klebba) [reply PUBLISHING.md]: good catch! fixt.

**Comment by theengineear** [comment]
All good on sorting out that `test` failure?

**Review by theengineear — DISMISSED** [review]
LGTM. Thanks!

**Review by theengineear — APPROVED** [review]
L _R_ GTM now. Thanks!

**Pull request merged**